pom.xml

```diff
@@ -30,12 +31,32 @@
    </scm>
    -->
 
+    <repositories>
+        <repository>
+            <id>bintray-jhult-maven</id>
+            <name>bintray-jhult-maven</name>
+            <url>https://dl.bintray.com/jhult/maven</url>
+        </repository>
+        <repository>
+            <id>bintray</id>
+            <name>JCenter</name>
+            <url>https://jcenter.bintray.com</url>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>bintray</id>
+            <name>JCenter</name>
+            <url>https://jcenter.bintray.com</url>
+        </pluginRepository>
+    </pluginRepositories>
+
```

Comment:
What is bintray?  is it publicly accessible?

Reply:
Bintray is publicly accessible. It contains a superset of Maven Central.

https://blog.bintray.com/2014/02/11/bintray-as-pain-free-gateway-to-maven-central/